say-scm-1.rockspec

```diff
@@ -0,0 +1,41 @@
+local package_name = "say"
+local package_version = "scm"
+local rockspec_revision = "1"
+local github_account_name = "lunarmodules"
+local github_repo_name = package_name
+local git_checkout = package_version == "scm" and "master" or package_version
+
+rockspec_format = "3.0"
+package = package_name
+version = package_version .. "-" .. rockspec_revision
+
+source = {
+  url = "git+https://github.com/"..github_account_name.."/"..github_repo_name..".git",
+  branch = git_checkout
+}
+description = {
+  summary = "Lua string hashing/indexing library",
+  detailed = [[
+    Useful for internationalization.
+  ]],
+  homepage = "https://lunarmodules.github.io/busted/",
+  license = "MIT <http://opensource.org/licenses/MIT>"
+}
+dependencies = {
+  "lua >= 5.1"
+}
+
+test_dependencies = {
+  "busted",
```

Comment:
luacov is also a dependency, unless it is removed from the `.busted` file

Reply:
Removed, so no it's not a test dependency only a CI runner dependency and the CI workflow can take care of it's own sauce.